conda/meta.yaml

```diff
@@ -37,10 +37,8 @@ requirements:
     - python
     - tbb-devel {{ tbb_devel }}
   run:
-    - graphlib-backport # for python < 3.9
     - numpy
     - python
-    - pyyaml
```

Comment:
Why is this not needed any more?

Reply:
I tried to find why it was there but could not. It might be from some ancient config?